api/src/refget/main.py

```diff
@@ -352,6 +497,7 @@ async def sequence(
     # Treat nonsensical requests first
     if end:
         if start > end:
+            LOG.info("Invalid client query with start > end")
             raise HTTPException(
                 status_code=501, detail="Circular regions not currently supported"
```

Comment:
Could you have the error message also say "you gave a range where start > end"? Users may not immediately realise that start > end implies circular regions.

Reply:
Done